src/dali_model.h

```diff
@@ -83,6 +82,19 @@ class DaliModel : public ::triton::backend::BackendModel {
     return output_order_;
   }
 
+  bool ShouldAutoCompleteConfig() const {
+    return should_auto_complete_config_;
+  }
+
+  TRITONSERVER_Error* AutoCompleteConfig() {
+    try {
+      AutofillConfig(model_config_, pipeline_inputs_, pipeline_outputs_, pipeline_max_batch_size_);
+      TRITON_CALL(SetModelConfig());
+    } catch (TritonError &err) {
+      return err.release();
```

Comment:
Out of curiosity: does this raw pointer need some freeing then? Who does it?

Reply:
It's eventually returned from a function called by Triton server and the server handles it